src/swr/print.ts

```diff
@@ -0,0 +1,42 @@
+import fs from "fs";
+import path from "path";
+import { CLIOptions } from "../cli";
+
+function printGeneratedTS(result: string, options: CLIOptions) {
+  fs.writeFileSync(path.join(options.outputDir, "index.ts"), result);
+}
+
+function printPackageJson(options: CLIOptions) {
+  const pkgJson = `
+  {
+    "name": "${options.name}",
+    "version": "${options.packageVersion}",
+    "description": "This package was generated by Rapini",
+    "main": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "license": "private",
+    "scripts": {
+      "build": "tsc index.ts --declaration --skipLibCheck --outDir dist"
+    },
+    "peerDependencies": {
+      "axios": "0.27.x",
+      "swr": "^1.3.0"
```

Comment:
Can it be more generic like `1.x.x` or `1.3.x`? I would rather not hardcode to a specific version like this

Reply:
[Semver](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) here. `1.x` and  `^1.3.0` are the same.